datadog/dogshell/dashboard_list.py

```diff
@@ -0,0 +1,201 @@
+        get_dashboards_parser.add_argument('dashboard_list_id', help="Dashboard list to show dashboards from")
+        get_dashboards_parser.set_defaults(func=cls._show_dashboards)
+
+        # Add Dashboards to Dashboard List parser
+        add_dashboards_parser = verb_parsers.add_parser('add_dashboards',
+            help="Add dashboards to an existing dashboard list")
+        add_dashboards_parser.add_argument('dashboard_list_id', help="Dashboard list to add dashboards to")
+        add_dashboards_parser.add_argument('dashboards',
+            help='A JSON list of dashboard dicts, e.g. ' +
+                 '[{"type": "custom_timeboard", "id": 1234}, {"type": "custom_screenboard", "id": 123}]')
+        add_dashboards_parser.set_defaults(func=cls._add_dashboards)
+
+        # Delete Dashboards from Dashboard List parser
+        delete_dashboards_parser = verb_parsers.add_parser('delete_dashboards',
+            help="Delete dashboards from an existing dashboard list")
+        delete_dashboards_parser.add_argument('dashboard_list_id', help="Dashboard list to delete dashboards from")
+        delete_dashboards_parser.add_argument('dashboards',
+            help='A JSON list of dashboard dicts, e.g. ' +
+                 '[{"type": "custom_timeboard", "id": 1234}, {"type": "custom_screenboard", "id": 123}]')
+        delete_dashboards_parser.set_defaults(func=cls._delete_dashboards)
+
+    @classmethod
+    def _post(cls, args):
+        api._timeout = args.timeout
+        format = args.format
+        name = args.name
+
+        res = api.DashboardList.create(name=name)
+        report_warnings(res)
+        report_errors(res)
+
+        if format == 'pretty':
+            print(pretty_json(res))
+        else:
+            print(json.dumps(res))
+
+    @classmethod
+    def _update(cls, args):
+        api._timeout = args.timeout
+        format = args.format
+        dashboard_list_id = args.dashboard_list_id
+        name = args.name
+
+        res = api.DashboardList.update(dashboard_list_id, name=name)
+        report_warnings(res)
+        report_errors(res)
+
+        if format == 'pretty':
+            print(pretty_json(res))
+        else:
+            print(json.dumps(res))
+
+    @classmethod
+    def _show(cls, args):
+        api._timeout = args.timeout
+        format = args.format
+        dashboard_list_id = args.dashboard_list_id
+
+        res = api.DashboardList.get(dashboard_list_id)
+        report_warnings(res)
+        report_errors(res)
+
+        if args.string_ids:
+            res['id'] = str(res['id'])
+
+        if format == 'pretty':
+            print(pretty_json(res))
+        else:
+            print(json.dumps(res))
+
+    @classmethod
+    def _show_all(cls, args):
+        api._timeout = args.timeout
+        format = args.format
+
+        res = api.DashboardList.get_all()
+        report_warnings(res)
+        report_errors(res)
+
+        if args.string_ids:
```

Comment:
I've chatted with @yannmh about this piece, it's definitely a legacy behavior, I think there is little benefit in adding more instances of it. Let's not include it.
I see it's used in `timeboard` `screenboard` `monitor` `downtime`, we can handle those separately.